developer/help/how-to-add-the-cmdlet-name-and-synopsis-to-a-cmdlet-help-topic.md

```diff
@@ -10,75 +10,26 @@ ms.assetid: 1d0e1eb1-a962-4406-9625-175cfa3364ad
 caps.latest.revision: 10
 ---
 # How to Add the Cmdlet Name and Synopsis to a Cmdlet Help Topic
+
 This section describes how to add content that is displayed in the NAME and SYNOPSIS sections of the cmdlet help. In the Help file, this content is added to the Command node for each cmdlet.
 
 > [!NOTE]
->  For a complete view of a Help file, open one of the dll-Help.xml files located in the Windows PowerShell installation directory. For example, the Microsoft.PowerShell.Commands.Management.dll-Help.xml file contains content for several of the Windows PowerShell cmdlets.
+> For a complete view of a Help file, open one of the dll-Help.xml files located in the Windows PowerShell installation directory. For example, the Microsoft.PowerShell.Commands.Management.dll-Help.xml file contains content for several of the Windows PowerShell cmdlets.
 
 ### To Add the Cmdlet Name and a Synopsis
 
-1.  Open the cmdlet Help file and locate the Command node for the cmdlet you are documenting. If you are adding a new cmdlet you will need to create a new Command node. Your Help file will contain a Command node for each cmdlet that you are providing Help content for.
-
```

Comment:
What happened to all of this content?

Reply:
This was an issue with my code fence logic and is fixed